src/main/java/uk/gov/hmcts/cmc/claimstore/services/legaladvisor/DocAssemblyClient.java

```diff
@@ -0,0 +1,137 @@
+                    .defendant(DocAssemblyRequest.Party.builder()
+                        .partyName(ccdCase.getRespondents()
+                            .get(0)
+                            .getValue()
+                            .getPartyName())
+                        .build())
+                    .judicial(DocAssemblyRequest.Judicial.builder()
+                        .firstName(userDetails.getForename())
+                        .lastName(userDetails.getSurname().orElse(""))
+                        .build())
+                    .currentDate(LocalDate.now())
+                    .hasFirstOrderDirections(
+                        directionList.contains(CCDOrderDirectionType.DOCUMENTS))
+                    .docUploadDeadline(
+                        ccdCase.getDocUploadDeadline())
+                    .hasSecondOrderDirections(
+                        directionList.contains(CCDOrderDirectionType.EYEWITNESS))
+                    .eyewitnessUploadDeadline(
+                        ccdCase.getEyewitnessUploadDeadline())
+                    .hasThirdOrderDirections(
+                        directionList.contains(CCDOrderDirectionType.MEDIATION))
+                    .docUploadForParty(
+                        ccdCase.getDocUploadForParty())
+                    .eyewitnessUploadForParty(
+                        ccdCase.getEyewitnessUploadForParty())
+                    .mediationForParty(
+                        ccdCase.getMediationForParty())
+                    .hearingIsRequired(
+                        ccdCase.getHearingIsRequired().toBoolean())
+                    .preferredCourtName(
+                        getCourtName(
+                            ccdCase.getHearingCourt()))
+                    .preferredCourtAddress(
+                        "this is an address EC2 BLA")// will be populated when the acceptance criteria are refined
+                    .estimatedHearingDuration(
+                        ccdCase.getEstimatedHearingDuration())
+                    .hearingStatement(
+                        ccdCase.getHearingStatement())
+                    .otherDirectionList(
+                        ccdCase.getOtherDirectionList().stream().map(
+                            direction -> CCDOrderDirection.builder()
+                                .extraOrderDirection(direction.getExtraOrderDirection())
+                                .otherDirection(direction.getOtherDirection())
+                                .sendBy(direction.getSendBy())
+                                .forParty(direction.getForParty())
+                                .build()
+                        ).collect(Collectors.toList()))
+                    .build()
+            )
+            .build();
+    }
+
+    // will be populated when the acceptance criteria are refined
+    private String getCourtName(CCDHearingCourtType ccdHearingCourtType) {
+        switch (ccdHearingCourtType) {
+            case EDMONTON:
```

Comment:
please change the description as part of Enum. So we have it in one centric place